README.md

```diff
@@ -64,3 +64,14 @@ This can be run with test explorer or pytest
 You can install pytest via pip
 To run pytest, simply run the following command from the root of your project.
 $ pytest
+
+To build the docker image, run either the development or the production build
```

Comment:
To make the code following this more readable I'd use backticks to make it markdown compliant (as used above)